events/lastAvatarStandingTrivia/entityScripts/triviaColorCheckBlue.js

```diff
@@ -10,36 +10,51 @@
 
 (function(){
     var _this,
-        bubble,
-        gameOn,
+        injector,
         gameZone;
     
     var DISQUALIFIED_POSITION = Script.require(Script.resolvePath('../clientScripts/triviaInfo.json')).DISQUALIFIED_POSITION,
         HALF_MULTIPLIER = 0.5,
-        ZONE_SQUARE_RADIUS = 1.5,
         RADIUS = 50,
-        RANGE = 5;
+        RANGE = 1000,
+        AUDIO_VOLUME = 0.5,
+        SOUND = SoundCache.getSound(Script.resolvePath('../entities/sounds/wrong-local.wav'));
 
     var ColorCheck = function(){
         _this = this;
     };
 
     ColorCheck.prototype = {
         preload: function(entityID){
+            console.log("color check blue loaded");
             _this.entityID = entityID;
             gameZone = Entities.getEntityProperties(
                 Entities.findEntitiesByName(
                     "Trivia Player Game Zone", MyAvatar.position, RADIUS)[0], ['position', 'rotation', 'dimensions']);
-            bubble = Entities.getEntityProperties(
-                Entities.findEntitiesByName("Trivia Bubble", MyAvatar.position, RADIUS)[0], ['visible']);
             if (Settings.getValue("activeTriviaColor") !== "Blue") {
+                console.log("my color was ", Settings.getValue("activeTriviaColor"), "which is not blue");
                 _this.ejectUser();
             }
             Script.setTimeout(function(){
                 _this.unload();
-            }, 15000);
+            }, 10000);
```

Comment:
what is this timeout for?

Reply:
to give everyone enough time to actually run the script before unloading it, assuming network latency means that not everyone gets it at the same time and that it should wait until all removals are finished.  A better way might be with a callback.

Reply:
got it. this assumption will come back to bite you at some point in the future! if it isn't already, could you please add a async callback as a todo in your readme?